changelogs/fragments/9847-Adding_audio_device-support_to_proxmox_kvm.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - audio - add missing audio hardware device handling for proxmox_kvm module (https://github.com/ansible-collections/community.general/issues/5192).
```

```suggestion
  - proxmox_kvm - add missing audio hardware device handling (https://github.com/ansible-collections/community.general/issues/5192).
```

Please refer to the standard formatting for the changelog fragments' entries:

https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to-format

The outcome of what you write will be published in the collection changelog:

https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html

---

Please also add the URL to the PR :)

---

Updated following recommandations, thanks